### PR TITLE
fix!: 🐛 Fixed TitleAlign Issue

### LINKED ISSUE #457: titleAlignment property does not work
**Describe the bug**
I am applying a TextAlign.center to the titleAlignment property and the Title is not centered.

**Expected behavior**
I would expect the title to appear centered.

Captura de pantalla 2024-06-27 a la(s) 11 06 08 p m

### PR DESCRIPTION
**Pull request by rashi-simform**
# Description
- Added `titleAlignment` and `descriptionAlignment` parameters to align title and description within the tooltip widget
- Renamed parameters `titleAlignment` to `titleTextAlign` and `descriptionAlignment` to `descriptionTextAlign`

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [ ] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [x] Yes, this PR is a breaking change.
- [ ] No, this PR is not a breaking change.

## Related Issues
Closes #457 

[Contributor Guide]: https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/blob/master/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ A Flutter package allows you to Showcase/Highlight your widgets step by step.
 
 ![The example app running in Android](https://raw.githubusercontent.com/SimformSolutionsPvtLtd/flutter_showcaseview/master/preview/showcaseview.gif)
 
+## Migration guide for release 4.0.0
+Renamed parameters `titleAlignment` to `titleTextAlign` and `descriptionAlignment` to `descriptionTextAlign` to correspond it more with the TextAlign property
```
Comment (aditya-chavda) [inline README.md:15]: Please also mention that the existing variables' behaviour/purpose has been changed.

File: lib/src/showcase.dart [hunk 2]
```diff
@@ -252,13 +254,25 @@ class Showcase extends StatefulWidget {
   /// Defaults to 7.
   final double toolTipSlideEndDistance;
 
+  /// Title alignment within tooltip widget
```
Comment (aditya-chavda) [inline lib/src/showcase.dart:257]: Can you please add widget word in there?
```suggestion
  /// Title widget alignment within tooltip widget
```

File: lib/src/tooltip_widget.dart [hunk 3]
```diff
@@ -439,44 +443,52 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
                                     : CrossAxisAlignment.center,
```
Comment (aditya-chavda) [inline lib/src/tooltip_widget.dart:443]: Please see if this is still required after the usage of `Align` as parent of the children.

File: lib/src/tooltip_widget.dart [hunk 4]
```diff
@@ -70,7 +72,7 @@ class ToolTipWidget extends StatefulWidget {
     required this.offset,
     required this.screenSize,
     required this.title,
-    required this.titleAlignment,
+    required this.titleTextAlign,
```
Comment (aditya-chavda) [inline lib/src/tooltip_widget.dart:75]: Can you please put this with the other text align and alignment property below?

**Review by aditya-chavda — CHANGES_REQUESTED** [review]

**Review by Sahil-Simform — APPROVED** [review]
LGTM!

**Review by aditya-css — APPROVED** [review]

**Pull request merged**